controller/eventQuestionController.js

```diff
@@ -0,0 +1,291 @@
+const HttpStatus = require('http-status-codes');
+const Mongoose = require('mongoose');
+
+const EventQuestion = require('../model/eventQuestionModel');
+const Event = require('../model/eventModel');
+const ValidationUtil = require('../util/validationUtil');
+const StatusEnum = require('../model/enums').StatusEnum;
+
+const create = (req, res) => {
+
+    const eventQuestion = new EventQuestion(req.body);
+
+    eventQuestion.save().then(() => {
+        return res.status(HttpStatus.CREATED).json(eventQuestion);
+    }).catch(err => {
+        const errorMessage = ValidationUtil.buildErrorMessage(err, 'create', 'eventQuestion');
+        return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
+            status: StatusEnum['ERROR'],
+            message: errorMessage
+        });
+    });
+
+};
+
+const getById = (req, res) => {
+
+    const eventQuestionId = req.params.id;
+    const user = req.decoded;
+
+    if (!ValidationUtil.isValidObjectId(eventQuestionId)) {
+        return res.status(HttpStatus.BAD_REQUEST).json({
+            status: StatusEnum['ERROR'],
+            message: 'Bad request'
+        });
+    }
+
+    if (!ValidationUtil.isUserAdmin(user.accessType)) {
+        return res.status(HttpStatus.UNAUTHORIZED).json({
+            status: StatusEnum['ERROR'],
+            message: 'Unauthorized'
+        });
+    }
+
+    EventQuestion.findById(eventQuestionId, { _id: 0 }).then(eventQuestion => {
+        if (!eventQuestion) {
+            return res.status(HttpStatus.NOT_FOUND).json({
+                status: StatusEnum['ERROR'],
+                message: 'Event question not found'
+            });
+        } else {
+            Event.findOne({ _id: new Mongoose.Types.ObjectId(eventQuestion.eventID), companyID: new Mongoose.Types.ObjectId(user.companyID) }, { _id: 0 }).then(event => {
```

Comment:
Not an error, but a suggestion. Comment on parts of your code so others can go through it with more ease when reviewing it. A simple explanation for situations like these where you went to find an event for the question to see if the event still exists before returning the event question would speed up the review process and make the code more readable.